packages/stylelint-config-ckeditor5/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "stylelint-config-ckeditor5",
+  "version": "2.0.0",
+  "description": "CKEditor 5 stylelint preset.",
+  "keywords": [
+    "stylelint",
+    "config",
+    "lint",
+    "ckeditor"
+  ],
+  "main": ".stylelintrc",
+  "dependencies": {
+    "stylelint-config-recommended": "^3.0.0"
+  },
+  "peerDependencies": {
+    "stylelint": ">=11.0.0"
```

Comment:
Version 12 has just been released. I know that his selector matches it, but let's put the latest version here.

Reply:
```suggestion
    "stylelint": ">=12.0.0"
```